.gitmodules

```diff
@@ -9,5 +9,5 @@
 	url = https://chromium.googlesource.com/linux-syscall-support
 [submodule "external/crashpad"]
 	path = external/crashpad
-	url = https://github.com/getsentry/crashpad.git
-	branch = getsentry
+	url = https://github.com/jpnurmi/crashpad.git
+	branch = feat/screenshot

```

Comment:
You don't need to change the submodule URL. As soon as you create PR onto our crashpad fork, the commits will become part of our object database, even if the branch never ends up in the repo. That means the only thing you need to do here is to update the submodule commit reference.